lib/between_meals/repo/git.rb

```diff
@@ -58,7 +58,7 @@ def last_author
       end
 
       def head_parents
-        @repo.head.target.parents
+        @repo.head.target.parents.map {|x| {:time => x.time}}
```

Comment:
It's not needed, not used anywhere as far as I can tell. This file needs refactoring btw, another sweep